src/api/python/CMakeLists.txt

```diff
@@ -186,30 +189,43 @@ add_custom_command(
   DEPENDS CVC5PythonicAPI
 )
 
-add_custom_target(
-  cvc5_python_api ALL DEPENDS
-    ${CVC5_PYTHON_BASE_LIB}
-    ${COPIED_PYTHONIC_FILES}
-    ${LICENSE_FILES}
-)
+if(ONLY_PYTHON_EXT_SRC)
 
-# figure out if we're in a virtualenv
-execute_process(OUTPUT_VARIABLE IN_VIRTUALENV
-  COMMAND
-  "${Python_EXECUTABLE}"
-  -c
-  "from __future__ import print_function; import os;
-print('YES' if 'VIRTUAL_ENV' in os.environ else 'NO', end='')")
-
-set(INSTALL_CMD
-    "${Python_EXECUTABLE} -m pip install ${CMAKE_CURRENT_BINARY_DIR}")
-# if we're in a virtualenv, we install it in the virtualenv lib location
-# otherwise install in prefix
-if ("${IN_VIRTUALENV}" STREQUAL "NO")
-  set(INSTALL_CMD "${INSTALL_CMD} --prefix ${CMAKE_INSTALL_PREFIX}")
-endif()
+  add_custom_target(
+    cvc5_python_api ALL DEPENDS
+      ${PYTHON_EXT_SRC_FILES}
+      ${COPIED_PYTHONIC_FILES}
+      ${LICENSE_FILES}
+  )
 
-message("Python bindings install command: ${INSTALL_CMD}")
+else()
+
+  add_custom_target(
+    cvc5_python_api ALL DEPENDS
+      ${CVC5_PYTHON_BASE_LIB}
+      ${COPIED_PYTHONIC_FILES}
+      ${LICENSE_FILES}
+  )
 
-install(CODE "execute_process(COMMAND ${INSTALL_CMD})"
-        FILE_PERMISSIONS OWNER_EXECUTE OWNER_WRITE OWNER_READ)
\ No newline at end of file
+  # figure out if we're in a virtualenv
+  execute_process(OUTPUT_VARIABLE IN_VIRTUALENV
+    COMMAND
+    "${Python_EXECUTABLE}"
+    -c
+    "from __future__ import print_function; import os; \
+     print('YES' if 'VIRTUAL_ENV' in os.environ else 'NO', end='')")
```

Comment:
We don't support Python2 anymore, so we shouldn't need to use __future__ for print here.

Reply:
Removed.